test/functional/sample_test.go

```diff
@@ -72,7 +71,7 @@ var _ = Describe("Samples", func() {
 	})
 	When("glance_v1beta1_glance_quota.yaml sample is applied", func() {
 		It("Glance is created", func() {
-			name := CreateGlanceFromSample("glance_v1beta1_glance_quota.yaml", glanceTest.Instance)
+			name := CreateGlanceFromSample("quotas/glance_v1beta1_glance_quota.yaml", glanceTest.Instance)
```

Comment:
@abays I fixed the envTest by preventing deletingLimits if no keystone CR is present in the ctlplane, and this actually solved the func test. I'm just moving the sample to a dedicated directory, and reference it from here (it can be done for any other sample file instead of keeping them in the root sample directory).
If it's ok w/ you (given @konan-abhi is off), you can re-apply the lgtm label and merge this.